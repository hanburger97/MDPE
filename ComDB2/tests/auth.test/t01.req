select 'user1'
set user user1
set password password1
put authentication on
select 'authentication turned on'
