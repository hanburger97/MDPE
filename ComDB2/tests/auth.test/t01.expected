('user1'='user1')
('authentication turned on'='authentication turned on')
